### PR TITLE
Assignment 3

### PR DESCRIPTION
**Pull request by jimflores5**

### CODE REVIEW

File: src/assignments/console-game.rst [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+#. User input (to make the program interactive).
+#. At least one loop and one conditional.
+#. Two or more functions, one of which must be a ``main()`` function.
+#. At least one list.
+#. A replay option (e.g. "Would you like to play again?").
+
+To boost your score for this assignment, you and your partner can include one
+or more of the following **Bonus Items**:
+
+#. Use one or more dictionaries in addition to (or in place of) the required
```
Comment (gildedgardenia) [inline src/assignments/console-game.rst:38]: I think it would be "dictionary(ies)"

File: src/assignments/console-game.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+Before you start coding, you and your partner must decide what game you want to
+create!
+
+#. Brainstorm ideas and write them down.
+#. Show the list of your ideas to your teacher. Receive feedback on which ones
+   are suitable for this assignment.
+#. Narrow down your list, then select the idea that excites you the most.
+
+Next, describe to each other what you want the program to do. Now is NOT the
+time to write code! Discuss the GAME, not the Python.
```
Comment (gildedgardenia) [inline src/assignments/console-game.rst:55]: "Discuss the GAME, not Python."

File: src/assignments/console-game.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+Before you submit your final program, be sure that your code:
+
+#. Works,
+#. Includes your name and your partner's name,
+#. Includes each of :ref:`the required items <assignment3-requirements>`,
+#. Survives invalid input. For example, if your program asks the user to enter
+   a number, it should not crash when they enter a letter.
+#. Includes enough comments to describe what each part of the code is doing.
+#. Follows proper Python naming conventions.
```
Comment (gildedgardenia) [inline src/assignments/console-game.rst:154]: I would end every point in a period.

**Review by gildedgardenia — APPROVED** [review]
This looks awesome!

File: src/assignments/console-game.rst (hunk 1 above)
Comment (jimflores5) [reply src/assignments/console-game.rst:38]: Fixed!

File: src/assignments/console-game.rst (hunk 2 above)
Comment (jimflores5) [reply src/assignments/console-game.rst:55]: Fixed.

File: src/assignments/console-game.rst (hunk 3 above)
Comment (jimflores5) [reply src/assignments/console-game.rst:154]: Done.

**Pull request merged**